pkg/controller/endpointslice/endpointslice_controller.go

```diff
@@ -141,7 +142,7 @@ func NewController(podInformer coreinformers.PodInformer,
 
 	c.endpointSliceLister = endpointSliceInformer.Lister()
 	c.endpointSlicesSynced = endpointSliceInformer.Informer().HasSynced
-	c.endpointSliceTracker = newEndpointSliceTracker()
+	c.endpointSliceTracker = endpointsliceutil.NewEndpointSliceTracker()
```

Comment:
I see you using the new one, but not removing the old one?

Reply:
@thockin  
I change old function name for public scope since we have moved the file `endpointslice_tracker.go` to new place. 
And I modfiy all the `newEndpointSliceTracker` function calling place